### DISCUSSION TITLE
cURL error 60: SSL certificate problem:

### DISCUSSION THREAD

**Discussion by bertux77** [tectalichq/public-openai-client-php#15] [General] [answered]
I'm trying to connect from laravel through your package in local mode but I can't get it to work. 
I get the following error:
`cURL error 60: SSL certificate problem: certificate has expired (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://api.openai.com/v1/completions`
Can anyone help me?
Thanks

**Comment by thejamescollins** [comment, marked as answer]
Hi there @bertux77,

It looks to me like the SSL certificate on `api.openai.com` is valid, and isn't due to expire until 28 April 2023:

https://www.ssllabs.com/ssltest/analyze.html?d=api.openai.com&latest

> Valid from Sat, 28 Jan 2023 00:46:58 UTC
> Valid until Fri, 28 Apr 2023 00:46:57 UTC (expires in 1 month and 24 days)

So things should be working.

Their status page (https://status.openai.com) shows some availability issues recently, so I'm not sure if that is the cause or not.

Is the issue still occurring today?

If you open https://api.openai.com/v1/completions in your web browser, do you get an SSL error?

James

**Reply by GitHub-Ahai** [reply]
laravel error : cURL error 28: Failed to connect to api.openai.com port 443: Connection timed out (see https://curl.haxx.se/libcurl/c/libcurl-errors.html) for https://api.openai.com/v1/audio/transcriptions ...

**Reply by GitHub-Ahai** [reply]
This is the latest method I can solve: https://github.com/dirk1983/chatgpt

**Reply by GitHub-Ahai** [reply]
I don't know why it can't be used in China
